CHANGELOG.md

```diff
@@ -5,6 +5,14 @@ All notable changes to this project will be documented in this file.
 The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
+## Unreleased
+
+## [0.0.16] - 2024-08-14
+
+- Add support for gasless transfers. Initially only supporting USDC sends on Base mainnet.
+- Add support for list historical balances for an asset of an address.
+- Add support for Ethereum-Mainnet and Polygon-Mainnet
+
```

Comment:
- Support for retrieving historical staking balances information
- USD value conversion details to the StakingReward object